editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1773,7 +1788,7 @@ export function getFrameChange(
   isParentFlex: boolean,
 ): PinOrFlexFrameChange {
   if (isParentFlex) {
-    return flexResizeChange(target, newFrame, null)
+    return flexResizeChange(target, 'flexBasis', 0) // KILLME
```

Comment:
What's with the `// KILLME` here? What needs killing and why?

Reply:
That was from the experiment originally.